### ISSUE TITLE
Look into pre-commit hooks

### ISSUE THREAD

**Issue by laurentboisvert** [coklacour/dbt_template#1]
Adding a simple `pre-commit` config could be extremely useful in this repository (or [modelisation](https://github.com/coklacour/modelisation)) to __standardize the code, avoid commiting "dangerous information", etc.

- For example, any .ipynb notebook commited could risk having sensible information stored in the notebook metadata (for when we ran the notebook - it could contain sensible client information). Not only that, but some outputs are extremely long and can clutter the repository. \* ideally you could use a `ipynb` to markdown converter and __never__ commit `.ipynb` files (like [Jupytext](https://github.com/mwouts/jupytext))

**Comment by coklacour** [comment]
You're absolutely right. I'll take care of it. Thanks @laurentboisvert for your input.
